package.json

```diff
@@ -57,7 +60,7 @@
   },
   "style": "dist/css/bootstrap.css",
   "sass": "scss/bootstrap.scss",
-  "main": "dist/js/bootstrap",
+  "main": "src/js/index",
```

Comment:
Another way would be lazily evaluate the modules with `import()` call, but not sure if that would either.